store/mongo/migration_2_0_0.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package mongo
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	mopts "go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	mstorev1 "github.com/mendersoftware/go-lib-micro/store"
+	mstore "github.com/mendersoftware/go-lib-micro/store/v2"
+)
+
+const (
+	findBatchSize = 255
+)
+
+type migration_2_0_0 struct {
+	client *mongo.Client
+	db     string
+}
+
+func (m *migration_2_0_0) Up(from migrate.Version) error {
+	ctx := context.Background()
+	client := m.client
+
+	collections := map[string]struct {
+		Indexes []mongo.IndexModel
+	}{
+		DevicesCollectionName: {
+			Indexes: []mongo.IndexModel{
+				{
+					Keys: bson.D{
+						{Key: mstore.FieldTenantID, Value: 1},
+						{Key: "_id", Value: 1},
+					},
+					Options: mopts.Index().
+						SetName(mstore.FieldTenantID + "_" + "_id"),
+				},
+			},
+		},
+		SessionsCollectionName: {
+			Indexes: []mongo.IndexModel{
+				{
+					Keys: bson.D{
+						{Key: mstore.FieldTenantID, Value: 1},
+						{Key: "device_id", Value: 1},
```

Comment:
Maybe declare "device_id" as a constant, similarly to dbFieldSessionID?

Reply:
makes sense. added.